ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -30,6 +30,12 @@ SD:
   - supply_air_heating_flowrate_capacity
   - cooling_thermal_power_capacity
   - supply_air_temperature_sensor
+  - zone_air_occupied_cooling_temperature_setpoint
+  - zone_air_unoccupied_cooling_temperature_setpoint
+  - zone_air_temperature_setpoint
+  - min_supply_air_flowrate_sensor
```

Comment:
the min of what? min and max are aggregations across multiple sensors.

Reply:
This is the min and max air flow rate values between which the vav can operate.

Reply:
high limit and low limit is what you should use then. see [`HLSAFS`](https://github.com/google/digitalbuildings/blob/accdc3b2685b08531cd83fcdd048f432bb2a2008/ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml#L4557) for an example.

Reply:
it also bounds the setpoint, not the sensor. looks like this might be a mistake here.

Reply:
Modified and used HLSAFS